src/main/java/net/openhft/chronicle/bytes/internal/AbstractBytesStore.java

```diff
@@ -53,6 +53,12 @@ public int hashCode() {
         return BytesStoreHash.hash32(this);
     }
 
+    @SuppressWarnings("rawtypes")
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
Why do we resurrect it here, and what's with hashCode()?

Reply:
Classes that overrides hashCode() should also override equals(). Hence, I have moved the declaration so they appear in pairs.